snare/tests/test_parse_tanner_response.py

```diff
@@ -73,6 +73,29 @@ def test_parse_type_two(self):
         expected_result = [self.expected_content, self.content_type, {}, 200]
         self.assertCountEqual(real_result, expected_result)
 
+    def test_parse_type_two_with_headers(self):
+
+        self.detection = {
+            "type": 2,
+            "payload": {
+                "page": "",
+                "value": "test.png",
+                "headers": {
+                    "content-type": "multipart/form-data"
+                }
+            }
+        }
+
+        async def test():
+            (self.res1, self.res2,
+             self.res3, self.res4) = await self.handler.parse_tanner_response(self.requested_name, self.detection)
+
+        self.loop.run_until_complete(test())
+        expected_header = {"content-type": "multipart/form-data"}
+        real_result = [self.res1, self.res2, self.res3, self.res4]
+        expected_result = [b'test.png', 'image/png', expected_header, 200]
```

Comment:
Use variables `self.expected_content`, `self.content_type` and assign after `detection`. Also I think to maintain consistency we can make `self.expected_header` and `self.expected_status_code` in setUp method.